### ISSUE TITLE
Chemical formulas read out as words

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molarity#215]
For https://github.com/phetsims/QA/issues/464. Similar in nature to https://github.com/phetsims/molecules-and-light/issues/255.
Found when testing iPad VO. When vowels were in the chemical formulas and could make pronounceable sounds, they would be read as words, rather than as individual letters. The most problematic is NO (as in nitrogen and oxygen) being read as "number."

**Comment by jessegreenberg** [comment]
Like https://github.com/phetsims/molecules-and-light/issues/255 I don't think there is anything we can do about this other than certain workarounds.
From https://www.levelaccess.com/zip-z-p-z-p-forcing-correct-pronunciation-screen-readers/

> Current solutions are either not elegant or not well-supported. Non-elegant solutions are generally not encouraged, however, below are some suggestions (when necessary) for forcing a pronunciation on a screen reader.

@terracoda is this something that PhET needs to address or can this be closed?

**Comment by terracoda** [comment]
I reviewed the A11y View and all the formulas look correct (proper capitalization and sub-scripted numbers). I also think they all sound prett good. Because they are in the PDOM, screen reader users can use letter-by-letter reading to to analyze the content if they need to.

When I looked at the code, I see we are not using an `abbr` tag. And I see teh subscripted "4" in the screenshot below is outside the quotations. Is that a problem? NUmbers, I suppose, don't need translation, so I think that's why, but just want to check.

![Screen Shot 2020-01-09 at 4 49 07 PM](https://user-images.githubusercontent.com/1902116/72101757-08455700-3300-11ea-8a6f-490105d93dcf.png)
 the A11y View, but I think everything is coded properly.

So, to sum up, we can investigate using an `abbr` tag, but I am pretty sure others have tested this and it doesn't help with pronunciation. It might help in other ways in terms of communication it is an abbreviation or be more future compliant, so we should investigate. This should not bock publication. I will open an issue in the A11y Research repo.

**Comment by terracoda** [comment]
I think when we investigate more with MathML, we can look into this deeper - there is a ChemML, but I have no idea if it is any good or supported by screen reader software. 

@jessegreenberg, after publication we can test the `abbr` tag, we have chemical names in Molecules and Light, so we can experiment there and leave this as a possible maintenance release if we discover a nice solution for MAL.

Molarity sound pretty to me.

@jessegreenberg, if there are no concerns about the number 4 being outside the quotation marks, can you please close this issue?

**Comment by terracoda** [comment]
Just adding a couple more links I saw, which I might look at more later.
- https://webaim.org/discussion/mail_thread?thread=7200
- https://en.wikipedia.org/wiki/Chemical_Markup_Language#Versioning

**Comment by terracoda** [comment]
@jessegreenberg, also, I think @twant may have tried the `abbr` tag, but I can't be sure.

**Comment by terracoda** [comment]
Re-posting my comment for wrapping up:

> @jessegreenberg, if there are no concerns about the number 4 being outside the quotation marks, can you please close this issue?

**Comment by jessegreenberg** [comment]
Thanks for inspecting @terracoda. I am not concerned about the quotations. I don't think it is about translation but more about how the dev tools display text. If the element is copied out of dev tools the quotes go away.
```html
<li data-trail-id="356-357-440-446-441-447-471-473-477" id="356-357-440-446-441-447-471-473-477" data-focusable="false" class="a11y-sibling" tabindex="-1">chemical formula of potassium chromate is K<sub tabindex="-1">2</sub>CrO<sub tabindex="-1">4</sub></li>
```

Going to make a note in the QA screen reader document that we don't expect the device to read chemical formulas correctly.

**Comment by jessegreenberg** [comment]
OK made note, closing.

**Comment by terracoda** [comment]
@jessegreenberg, thanks for letting me know that the quotes are from devtools!

**Comment by jessegreenberg** [comment]
Well, its my best guess anyway!
